tests/Integration/OtherTest.php

```diff
@@ -148,6 +148,51 @@ public function test_load_js_tracker(): void {
 		wp_print_scripts();
 		$output = ob_get_clean();
 
+		self::assertSame(
+			"<script type='text/javascript' data-parsely-site=\"blog.parsely.com\" src='https://cdn.parsely.com/keys/blog.parsely.com/p.js?ver=" . \Parsely::VERSION . "' id=\"parsely-cfg\"></script>\n",
+			$output,
+			'Failed to confirm script tag was printed correctly'
+		);
+	}
+
+	/**
+	 * Test the tracker script enqueue.
+	 *
+	 * @covers \Parsely::load_js_tracker
+	 * @uses \Parsely::get_asset_cache_buster
+	 * @uses \Parsely::api_key_is_missing
+	 * @uses \Parsely::api_key_is_set
+	 * @uses \Parsely::get_options
+	 * @uses \Parsely::post_has_trackable_status
+	 * @uses \Parsely::register_js
+	 * @uses \Parsely::script_loader_tag
+	 * @uses \Parsely::update_metadata_endpoint
+	 * @group insert-js
+	 */
+	public function test_load_js_tracker_with_cloudflare(): void {
+		add_filter( 'wp_parsely_enable_cfasync_tag', '__return_true' );
```

Comment:
```suggestion
		add_filter( 'wp_parsely_enable_cfasync_attribute', '__return_true' );
```

The `data-cfasync` is an attribute, not a tag.

But, instead of being so granular, perhaps we could call it `wp_parsely_enable_cloudflare_support` so that it's easier to remember, more self-descriptive, and allows the same filter to also power any future Cloudflare adaptations that are needed.

Reply:
Agreed on naming it `wp_parsely_enable_cfasync_attribute`. 

However, I'd keep that instead of something more Cloudflare generic. That is for two situations that might arise in the future: 

- Cloudflare releases a new feature and a user wants to have `cfasync` enabled but that feature disabled.
- A party other than Cloudflare starts using the `cfasync` attribute. 

If you agree, I'd apply your suggestion as is (and change it in other occurrences in this PR's code).

Reply:
OK, keep it as `wp_parsely_enable_cfasync_attribute` for now then.